### PR TITLE
Vuex Clients

### PR DESCRIPTION
**Pull request by VPLEV23**

### CODE REVIEW

File: src/store/client.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      first_Name: "",
+      second_Name: "",
+      phone: ""
+    }
+  },
+  getters: {
+    allClients: state => {
+      return state.clients;
+    },
+    allClient: state => {
```
Comment (user4i) [inline src/store/client.js]: not needed?

File: src/store/client.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import Vue from "vue";
+import axios from "axios";
+const baseUrl = "http://localhost:3000/clients";
+export default {
+  state: {
+    isFormHidden: true,
+    clients: [],
+    client: {
```
Comment (user4i) [inline src/store/client.js]: ? not needed now

File: src/store/client.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    ADD_CLIENT(state, payload) {
+      state.clients.push(payload);
+    },
+    SET_CLIENT(state, payload) {
+      Vue.set(state, "clients", ...[payload]);
+    }
+  },
+  actions: {
+    SET_CLIENT({ commit }, payload) {
```
Comment (user4i) [inline src/store/client.js]: ? not needed

File: src/store/client.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  getters: {
+    allClients: state => {
+      return state.clients;
+    },
+    allClient: state => {
+      return state.client;
+    }
+  },
+  mutations: {
+    SAVE_CLIENT(state, payload) {
```
Comment (user4i) [inline src/store/client.js]: ? not needed

**Review by user4i — CHANGES_REQUESTED** [review]

File: src/store/client.js (hunk 1 above)
Comment (VPLEV23) [reply src/store/client.js]: done

File: src/store/client.js (hunk 2 above)
Comment (VPLEV23) [reply src/store/client.js]: done

File: src/store/client.js (hunk 3 above)
Comment (VPLEV23) [reply src/store/client.js]: done

File: src/store/client.js (hunk 4 above)
Comment (VPLEV23) [reply src/store/client.js]: done

**Pull request merged**